### PR TITLE
Feature/availability request register booking in BE

### PR DESCRIPTION
**Pull request by silksil**
- create `useCheckout` hook for storing everything in relation to group booking
- conduct group booking request to BE
- changes to group booking form
-  Easily send notifications to the user, e.g.

TODO: the `useCheckout` hook is currently only working for groupBooking, but still needs to be made suitable for a "normal" booking. 
`` `
  const { enqueueSnackbar } = useSnackbar();
       enqueueSnackbar('Please select more then 10 rooms to continue', {
        variant: 'error'
      });
      
```

### CODE REVIEW

File: src/store/actions.ts [hunk 1]
```diff
@@ -103,10 +106,24 @@ export interface SetWalletAuthAction {
   payload?: BookingsAuthResponse;
 }
 
+export interface SetWalletAuthAction {
+  type: 'SET_WALLET_AUTH';
+  payload?: BookingsAuthResponse;
+}
```
Comment (bobetbat) [inline src/store/actions.ts:112]: duplicate SetWalletAuthAction

File: src/containers/OrgDetails.tsx [hunk 2]
```diff
@@ -107,71 +142,53 @@ export const OrgDetails = () => {
...
-          ...groupCheckout,
-          organizerInfo: {
-            ...groupCheckout?.organizerInfo,
-            ...values
-          }
-        } as GroupCheckOut
-      });
+      const { privacy: _, ...rest } = values;
+      const formattedOrganizerInfo = getFormattedOrganizerInfo(
+        rest as OrganizerInformationForm
```
Comment (bobetbat) [inline src/containers/OrgDetails.tsx:160]: `as` should not be used it is a workaround, functions that returns values should have output typed

**Review by bobetbat — COMMENTED** [review]
some small fixes required

**Review by dave-ok — COMMENTED** [review]
Please check out the following comments

**Pull request merged**